ext/DynaLoader/DynaLoader_pm.PL

```diff
@@ -292,12 +298,14 @@ sub croak   { require Carp; Carp::croak(@_)   }
 
 sub bootstrap_inherit {
     my $module = $_[0];
-    local *isa = *{"$module\::ISA"};
-    local @isa = (@isa, 'DynaLoader');
+
+    no strict 'refs';
+    push @{"$module\::ISA"}, 'DynaLoader';
     # Cannot goto due to delocalization.  Will report errors on a wrong line?
     bootstrap(@_);
 }
 
+our (@args, $module, @dirs, $file);
```

Comment:
This declaration probably makes more sense inside sub bootstrap?

Reply:
I couldn't decide if any of these variables were needed so I left them in the stash. Happy to move them into the sub.

Reply:
If I put them in sub bootstrap, then they're still `@DynaLoader::args`, right? Are we just discussing style or a functional difference here? Not the local statements following this line.

Reply:
The lexical alias would only exist in the scope of the sub. It would alias to the same package variables either way.

Reply:
ok.